src/main/java/racingcar/validator/InputValidator.java

```diff
@@ -0,0 +1,52 @@
+package racingcar.validator;
+
+public class InputValidator {
+
+    public static void validatePlayer(String strPlayer) {
+        if (strPlayer == null || strPlayer.isEmpty()) {
```

Comment:
```suggestion
        if (strPlayer == null || strPlayer.isBlank()) {
```
isBlank 로 하면 " " 공백도 잡아줘서 더 안전할것 같아요!

Reply:
아 isBlank()가 있군요!!